public/js/modules.js

```diff
@@ -1507,7 +1507,7 @@ function Splitter(x, y, scope = globalScope, dir = "RIGHT", bitWidth = undefined
     CircuitElement.call(this, x, y, scope, dir, bitWidth);
     this.rectangleObject = false;
 
-    this.bitWidthSplit = bitWidthSplit || prompt("Enter bitWidth Split").split(' ').map(function(x) {
+    this.bitWidthSplit = bitWidthSplit || prompt("Enter bitWidth Split").split(' ').filter(x => x != '').map(function(x) {
```

Comment:
Looks good but can you can try and use`.trim().split(' ')` instead maybe?

Reply:
Oh, my bad, I think the thing you are trying to do, won't be fixed by this, it can better be fixed by regex but this is good enough.